.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -6,10 +6,18 @@
       <scope name="Production" level="ERROR" enabled="false" />
       <scope name="Production minus fixtures" level="ERROR" enabled="true" />
     </inspection_tool>
+    <inspection_tool class="MalformedFormatString" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
Was this change intentional?

Reply:
I was messing with Intellij settings; I'm used to these project files being in the `.gitignore` - I'll remove it